assets/css/edd-admin.css

```diff
@@ -2652,9 +2652,11 @@ body.download_page_edd-reports {
 #edd-reports-tiles-wrap {
 	display: flex;
 	flex-wrap: wrap;
+	justify-content: center;
 	margin-left: 10px;
 	margin-right: 10px;
```

Comment:
What about removing all of the margins, so they would be zero, and adjusting the padding to 20px 10px, so that the room at the top looks less cramped?

Note: if the number of tiles are not divisible by 3, the last row tiles will be centered (because of my justify-content suggestion), which I'm okay with, but does bring them out of alignment with the grid.

Reply:
@spencerfinnell ?  :point_up:

Reply:
What about using actual CSS grid to make a true grid? 

https://www.w3.org/TR/css3-grid-layout/
https://caniuse.com/css-grid

It seems appropriate for a use case like this.